src/pages/Home/mapbox-gl-geocoder.css

```diff
@@ -0,0 +1,242 @@
+/* Basics */
+.mapboxgl-ctrl-geocoder,
+.mapboxgl-ctrl-geocoder *,
+.mapboxgl-ctrl-geocoder *:after,
+.mapboxgl-ctrl-geocoder *:before {
+  box-sizing: border-box;
+}
+
+.mapboxgl-ctrl-geocoder {
+  font-size: 18px;
+  line-height: 24px;
+  font-family: "Open Sans", "Helvetica Neue", Arial, Helvetica, sans-serif;
+  position: relative;
+  background-color: #fff;
+  width: 100%;
+  min-width: 240px;
+  z-index: 1;
+  border-radius: 4px;
+  transition: width .25s, min-width .25s;
+}
+
+.mapboxgl-ctrl-geocoder--input {
+  font: inherit;
+  width: 100%;
+  border: 0;
+  background-color: transparent;
+  margin: 0;
+  height: 50px;
+  color: #404040; /* fallback */
```

Comment:
Do we need to add fall back color if create-react-app already uses prefixes? [what is prefixes?](https://www.lifewire.com/css-vendor-prefixes-3466867)

Reply:
Good question - that CSS is from the third party. I know prefixes are still needed for transition properties, but I would have to check into the others.

Reply:
The fall back color in this case is used to make sure text remains visible while backgrounds are loading. It is not a prefix issue.